cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/WebServerConfig.java

```diff
@@ -253,6 +253,29 @@
   private static final String WEBSERVER_SSL_EXCLUDE_PROTOCOLS_DOC = "Sets the excluded protocols (list of patterns) for the webserver. "
       + "If not set, it does not change the system defaults.";
 
+  /**
+   * <code>webserver.ssl.sts.enabled</code>
+   */
+  public static final String WEBSERVER_SSL_STS_ENABLED = "webserver.ssl.sts.enabled";
+  public static final boolean DEFAULT_WEBSERVER_SSL_STS_ENABLED = false;
+  public static final String WEBSERVER_SSL_STS_ENABLED_DOC = "Enables the Strict Transport Security header in the web servers responses.";
+
+  /**
+   * <code>webserver.ssl.sts.includeSubDomains</code>
+   */
+  public static final String WEBSERVER_SSL_STS_INCLUDESUBDOMAINS = "webserver.ssl.sts.includeSubDomains";
+  public static final boolean DEFAULT_WEBSERVER_SSL_STS_INCLUDESUBDOMAINS = true;
+  public static final String WEBSERVER_SSL_STS_INCLUDESUBDOMAINS_DOC = "Sets the includeSubDomains directive of the STS header. "
+    + "Only effective when webserver.ssl.sts.enabled is true.";
```

Comment:
Nits: Can we use all lowercase in config names for consistency? -- i.e. `webserver.ssl.sts.includeSubDomains` -> `webserver.ssl.sts.includesubdomains` or `webserver.ssl.sts.include.subdomains`?